### ISSUE TITLE
neobundle#load_cache() cannot detect changes in TOML file

### ISSUE THREAD

**Issue by iTakeshi** [Shougo/neobundle.vim#469]
I'm using a TOML file to manage plugins with neobundle, but when I make changes in the TOML file, `neobundle#load_cache()` returns `0` and the change is not reflected.
This is because `neobundle#load_cache()` checks only the ftime of $MYVIMRC, as mentioned in #383.
Do you have any suggestion handling this situation?

The minimum `init.vim` is like following:
```vim
set runtimepath+=~/.config/nvim/bundle/neobundle.vim
call neobundle#begin('~/.config/nvim/bundle/')

if neobundle#load_cache()
  echomsg "cache refreshed"
  call neobundle#load_toml('plugin.define.toml')
  NeoBundleSaveCache
else
  echomsg "using cached config"
endif

call neobundle#end()

NeoBundleInstall
NeoBundleList
```

and the TOML file is:
```toml
[[plugins]]
repository = 'Shougo/neobundle.vim'
fetch = 1

# try to comment in/out following 2 lines and reboot nvim
[[plugins]]
repository = 'Shougo/unite.vim'
```
my environment is: ubuntu 14.04, neovim 0.1.1 (installed using apt-get), and the latest neobundle.vim.
`:version` says:
```
NVIM 0.1.1 (compiled Dec 11 2015 15:23:26)
Build type: RelWithDebInfo
Compilation: /usr/bin/x86_64-linux-gnu-gcc -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -Wconversion -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -g -DDISABLE_LOG -Wall
-Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wvla -fstack-protector --param ssp-buffer-size=4 -DINCLUDE_GENERATED_DECLARATIONS -DHAVE_CONFIG_H -I/build/neovim-GUOHk_/neovim-0.1.0
ubuntu1+git201512100819+2042+15~ubuntu14.04.1/build/config -I/build/neovim-GUOHk_/neovim-0.1.0ubuntu1+git201512100819+2042+15~ubuntu14.04.1/src -I/build/neovim-GUOHk_/neovim-0.1.0ubuntu1+git201512100819+20
42+15~ubuntu14.04.1/.deps/usr/include -I/build/neovim-GUOHk_/neovim-0.1.0ubuntu1+git201512100819+2042+15~ubuntu14.04.1/.deps/usr/include -I/usr/include/luajit-2.0 -I/build/neovim-GUOHk_/neovim-0.1.0ubuntu1
+git201512100819+2042+15~ubuntu14.04.1/.deps/usr/include -I/build/neovim-GUOHk_/neovim-0.1.0ubuntu1+git201512100819+2042+15~ubuntu14.04.1/.deps/usr/include -I/build/neovim-GUOHk_/neovim-0.1.0ubuntu1+git201
512100819+2042+15~ubuntu14.04.1/.deps/usr/include -I/build/neovim-GUOHk_/neovim-0.1.0ubuntu1+git201512100819+2042+15~ubuntu14.04.1/.deps/usr/include -I/usr/include -I/build/neovim-GUOHk_/neovim-0.1.0ubuntu
1+git201512100819+2042+15~ubuntu14.04.1/build/src/nvim/auto -I/build/neovim-GUOHk_/neovim-0.1.0ubuntu1+git201512100819+2042+15~ubuntu14.04.1/build/include
Compiled by buildd@lgw01-39

Optional features included (+) or not (-): +acl   +iconv    +jemalloc
For differences from Vim, see :help vim-differences

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"
```

**Comment by iTakeshi** [comment]
Also, this situation is same if I use `plugin.define.vim` instead of the TOML file.
```vim
NeoBundleFetch 'Shougo/neobundle.vim'

" comment in/out following line and reboot nvim
NeoBundle 'Shougo/unite.vim'
```

**Comment by NigoroJr** [comment]
:+1:

**Comment by Shougo** [comment]
Fixed by #470
